extensions/console-notification.crd.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Any value in `extensions/examples/` that we could keep usage examples in?  Something like a single file you `oc create -f` on to create a single notification.

Reply:
Good idea.  This CRD file is ultimately going to not live here, so we'll have to figure out what to call the folder containing examples.  `extensions-examples`?  That sort of terrible.  Ideas?

Reply:
This is probably less important now that `frontend/public/models/yaml-templates.ts` was updated with example yaml.

![Screen Shot 2019-03-13 at 4 29 27 PM](https://user-images.githubusercontent.com/895728/54312371-31486e80-45ad-11e9-8240-3efc140203ab.png)